org.envirocar.app/res/layout/activity_signin.xml

```diff
@@ -95,7 +95,8 @@
             app:layout_constraintEnd_toEndOf="@id/activity_signin_guideline_right"
             app:layout_constraintStart_toStartOf="@id/activity_signin_guideline_left"
             app:layout_constraintTop_toBottomOf="@+id/activity_signin_username_input_layout"
-            app:layout_constraintWidth_max="500dp">
+            app:layout_constraintWidth_max="500dp"
+            app:passwordToggleEnabled="true">
```

Comment:
This looks fine!
but it will generate issue of overlapping of error icon and password toggle.
Try to resolve that also.
If you need help, check my PR how to fix it.

Reply:
Okay bro

Reply:
Fixed the overlapping issue.
Please review